### DISCUSSION TITLE
wazuh logs show in nsm but not kibana

### DISCUSSION THREAD

**Discussion by DinhDiesel** [Security-Onion-Solutions/securityonion#5947] [Unsupported Versions] [answered]
I still cant seem to get the windows event logs indexed. successfully installed wazuh agent & sysmon. I've opened up 1514/1514 for regisration and adding agent process directed agents to manager and imported the keys to appropriate agents still dont see any indexing....also added host IPs where agents are running on so-allow wazuh API. any assistance would be appreciated! 

-currently running 2.3.8 distributed (manager & heavy)

**Comment by dougburks** [comment, marked as answer]
If your manager is just a manager and not a managersearch, then there may not be anywhere for those logs to go. Heavy nodes only index their own logs and not any logs from the manager. So if you want to send Wazuh logs to the manager it would either need to be a managersearch or you would need a separate search node. For more information, please see:
https://docs.securityonion.net/en/2.3/architecture.html
